readme.txt

```diff
@@ -103,6 +94,46 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 == Changelog ==
 
+= 5.8 =
+
+* Release date: February 6, 2018
+* Release post: https://wp.me/p1moTy-731
+
+**Major Enhancements**

```

Comment:
Add search to Major Enhancements:

```
Search: Release from being in beta. Many improvements: customizer UX improvements; custom sorting support; post type filtering; and bug fixes.
```

Reply:
@gibrown I'm still working on the changelog items and testing list for Search ([quite a few more PRs to go through](https://github.com/Automattic/jetpack/pulls?page=1&q=is%3Apr+is%3Aclosed+label%3A%22%5BStatus%5D+Needs+Changelog%22&utf8=%E2%9C%93)). I'd be happy to get your feedback on it when it's ready! I've just committed what I have so far, and will continue to add to it later today. Feel free to add to the testing list as well, or ping me.

Thank you!

Reply:
@jeherve np, sorry for jumping the gun. Came across it looking for something else. :)